packages/core/src/utilities/text-decoration/_text-decoration.docs.scss

```diff
@@ -3,15 +3,9 @@ Text decoration
 
 Set whether text is underlined or not using the `text-decoration` utility.
 
-**Format**: `ds-u-text-decoration--[VALUE]`
```

Comment:
Maybe consider grouping text alignment, decoration, and transform and possible word break under a new `text styles` page?

Reply:
same thought here: https://github.com/CMSgov/design-system/pull/554#discussion_r347431092